support/routes/v2/formsg/formsgGGsRepair.ts

```diff
@@ -41,7 +41,11 @@ const REQUESTER_EMAIL_FIELD = "Email"
 const OPTION_TO_SUBMIT_CSV = "Do you want to upload list of sites as a CSV?"
 const ATTACHMENT = "Attachment"
 const REPO_NAME_FIELD = "Table (Repo Name (in GitHub))"
+// const HAS_REPO_BEEN_LOCKED = "Have you locked the repo?"
```

Comment:
Should remove?

Reply:
rm-ed [8aeca25](https://github.com/isomerpages/isomercms-backend/pull/1327/commits/8aeca25476c5ed4ef92500642919bfb89e80987d)